packages/kbn-field-types/src/kbn_field_types.ts

```diff
@@ -49,3 +48,18 @@ export const castEsToKbnFieldTypeName = (esType: ES_FIELD_TYPES | string): KBN_F
  */
 export const getFilterableKbnTypeNames = (): string[] =>
   registeredKbnTypes.filter((type) => type.filterable).map((type) => type.name);
+
+export function normalizeType(type: string) {
```

Comment:
ℹ️ I just moved this helper here as we are duplicating this code in many places.

Reply:
Yeah this is fair, I just moved it and didnt think to rename. I addressed it here https://github.com/elastic/kibana/pull/174677/commits/e843fec0be2987cb520e2e538a2f4d8ae8b363c9